groups/groups.yaml

```diff
@@ -277,6 +279,7 @@ groups:
     settings:
       ReconcileMembers: "true"
     members:
+      - k8s-infra-release-admins@kubernetes.io
```

Comment:
I would like to understand more about why membership in this group
- what does it allow?
- why is that needed?

Reply:
Grants [`roles/containeranalysis.occurrences.viewer`](https://cloud.google.com/container-registry/docs/ca-access-control#iam_roles), which I don't think is strictly required here. I've dropped it from the PR.

ref: https://github.com/kubernetes/k8s.io/blob/ed671b680ce43ecc024d56df224bb85a8ce1a36e/infra/gcp/ensure-prod-storage.sh#L238-L245